integration/src/main/java/scripts/client/RawlsClient.java

```diff
@@ -39,6 +42,12 @@ private WorkspacesApi createWorkspacesApi(String basePath, TestUserSpecification
       throws IOException {
     var workspaceApiClient =
         new ApiClient() {
+          @Override
+          protected void performAdditionalClientConfiguration(ClientConfig clientConfig) {
+            super.performAdditionalClientConfiguration(clientConfig);
+            clientConfig.connectorProvider(new JdkConnectorProvider());
+          }
+
```

Comment:
Did this change end up fixing the issue we were facing?

Reply:
Yes it did!